protobuf_to_dict/convertor.py

```diff
@@ -192,25 +195,33 @@ def _get_field_mapping(pb, dict_value, strict):
     return field_mapping
 
 
-def _dict_to_protobuf(pb, value, type_callable_map, strict, ignore_none):
+def _dict_to_protobuf(pb, value, type_callable_map, strict, ignore_none, ignore_values):
```

Comment:
What do you think about converting ignore_values into set if it is not already a set. That way if somebody passed a list, we still use the set for contain operation.